.changeset/curvy-ties-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": patch
+---
+
+- **docs**: Update tailwind integration guide. By @kyhyco #636
```

Comment:
I think we will need this guy [chore: upgrade packemon to v4](https://github.com/coinbase/onchainkit/pull/616) or at least update all the `lib` to `esm` in the main package.json file.

Docs `yarn build` is looking for `lib/index.js`. But the problem is OCK `yarn build` does not produce a `lib` folder at the moment.

Reply:
Ah!

Reply:
You right, we don't produce a `lib` folder.

Reply:
Ok, tomorrow morning (in 7 hours) I am going to give a try merging your PRs.